iRonCub-Mk3/CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ option(INSTALL_ALL_IRONCUB_VERSIONS "Install all variations of iRonCub robots."
 
 if(INSTALL_ALL_IRONCUB_VERSIONS)
     set(robot_list iRonCub-Mk3 iRonCub-Mk3-FT-ARMS) #Add to the list other iRonCub3 versions (that share the same configuration files)
+    set(robot_list iRonCub-Mk3 iRonCub-Mk3-WALKING-TURBINES) #Add to the list other iRonCub3 versions (that share the same configuration files)
```

Comment:
Why didn't you append the list in the lin on top?

Not sure if this will overwrite the one defined above.

Reply:
Something like `set(robot_list iRonCub-Mk3 iRonCub-Mk3-FT-ARMS iRonCub-Mk3-WALKING-TURBINES)`

Reply:
Because I made a mistake; now I added the installation also for `iRonCub-Mk3-WALKING-NO-TURBINES` as you suggested.